libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -586,7 +586,7 @@ bool ExpressionCompiler::visit(FunctionCall const& _functionCall)
 			m_context << Instruction::CREATE;
 			// Check if zero (out of stack or not enough balance).
 			m_context << Instruction::DUP1 << Instruction::ISZERO;
-			m_context.appendConditionalInvalid();
+			m_context.appendConditionalRevert();
```

Comment:
This is part of contract creation (also the below cases, where transfer failed, code not present). I think the original agreement was these are invalid as they are not part of input validation.

Maybe we need more clarified rules.